config/jobs/kubernetes/kubeadm/OWNERS

```diff
@@ -0,0 +1,15 @@
+# See the OWNERS file documentation:
+#  https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md
+
+approvers:
+- luxas
+- timothysc
+- fabriziopandini
+- neolit123
+reviewers:
```

Comment:
When approvers and reviewers are the same you can drop the reviewers section.

Reply:
thanks, good to now.
will possibly clean these in future PRs that touch this space.